metricbeat/module/ceph/mgr_cluster_health/_meta/data.json

```
@@ -0,0 +1,31 @@
+{
+    "@timestamp": "2017-10-12T08:05:34.853Z",
+    "agent": {
+        "hostname": "host.example.com",
+        "name": "host.example.com"
+    },
+    "ceph": {
+        "mgr_cluster_health": {
```

As mentioned in other comment, it'd be nice if we could use the same fields as old modules for common data. Here for example it could be `cluster_health`.
```suggestion
        "cluster_health": {
```

---

Unfortunately this is also metricset name, hence it can't be replaced (causes "metricset name already registered" error).

---

The name of the metricset doesn't need to be neccesarily in the event, and in this case I think it would be beneficial to have equivalent fields in the same places.

To do that we can report events with the metrics at the module level:
```
reporter.Event(mb.Event{
    ModuleFields: common.MapStr{
      "cluster_health": event,
    },
})
```

Or even at the root level:
```
reporter.Event(mb.Event{
    RootFields: common.MapStr{
      "ceph": common.MapStr{
        "cluster_health": event,
       },
    },
})
```

If, for some reason, it is needed to differentiate per metricset in a query or dashboard, the `event.dataset` field can be used.